src/nginx_prometheus_exporter.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2024 Canonical
+# See LICENSE file for licensing details.
+"""Nginx Prometheus exporter workload."""
+
+import logging
+
+from nginx import NGINX_PORT
+from ops.pebble import Layer
+
+logger = logging.getLogger(__name__)
+
+NGINX_PROMETHEUS_EXPORTER_PORT = "9113"
```

Comment:
This is unused in this file.
Is it possible to pass it as a cli arg to the exporter? Seems less brittle to have it explicit in the layer def.

Reply:
I placed it there since it is an nginx prometheus exporter default.
We only use it in charm.py.

We can use `--web.listen-address=:9113` in the exporter, but it is redundant...

Reply:
I think specifying `f"--web.listen-address=:{NGINX_PROMETHEUS_EXPORTER_PORT}"` would be good practice because defaults may change (unlikely, but still), and we have code that relies on this.